parameter_files/ProjectParameters_PA2024CH.yml

```diff
@@ -0,0 +1,74 @@
+default:
+
+    paths:
+        data_location_ext: ../pacta-data/2023Q4/
+
+    reporting:
+        project_report_name: PA2024CH
+        display_currency: CHF
+        currency_exchange_value: 1.190405
+
+    parameters:
+        timestamp: 2023Q4
+        dataprep_timestamp: 2023Q4
+        start_year: 2023
+        horizon_year: 5
+        select_scenario: WEO2023_NZE_2050
+        scenario_other: GECO2023_1.5C-Unif
+        portfolio_allocation_method: portfolio_weight
+        scenario_geography: Global
+
+    methodology:
+        has_map: TRUE
+        has_revenue: FALSE
+        has_credit: FALSE
+        inc_emissionfactors: TRUE
+
+    sectors:
+        tech_roadmap_sectors:
+            - Power
+            - Automotive
+            - Oil&Gas
+            - Coal
+        pacta_sectors_not_analysed:
+            - Steel
+            - Aviation
+            - Cement
+        green_techs:
+            - RenewablesCap
+            - HydroCap
+            - NuclearCap
+            - Hybrid
+            - Electric
+            - FuelCell
+            - Electric Arc Furnace
+        alignment_techs:
+            - RenewablesCap
+            - CoalCap
+            - Coal
+            - Oil
+            - Gas
+            - Electric
+            - ICE
+
+    asset_types:
+        - Equity
+        - Bonds
+        - Others
+        - Funds
```

Comment:
not sure if these are ever used

Reply:
This gets read in by `pacta.portfolio.utils::set_project_parameters()` as `asset_types` here:
https://github.com/RMI-PACTA/pacta.portfolio.utils/blob/6f2158548b552fa4204179eb5bf49974f3a93ccf/R/set_project_parameters.R#L51C14-L51C25

Reply:
These appear to be leftover from the old TM code and we're only used for 
https://github.com/search?q=repo%3ARMI-PACTA%2FPACTA_analysis%20asset_types&type=code

and were only used by some `website_text()` function:
https://github.com/RMI-PACTA/PACTA_analysis/blob/9f760b8319791db07dc07317ea844965980b181f/0_web_functions.R#L272-L303

Reply:
I will remove these